pkg/cli/upgradeassistant/internal/repository/models/emailhost.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type EmailHost struct {
+	ID        primitive.ObjectID `bson:"_id,omitempty"           json:"id,omitempty"` // 主键
+	Name      string             `bson:"name"       json:"name"`
+	Port      int                `bson:"port"       json:"port"`
+	Username  string             `bson:"username"   json:"username"`
+	Password  string             `bson:"password"   json:"password"`
+	IsTLS     interface{}        `bson:"is_tls"     json:"isTLS"`
```

Comment:
Why does json tag use `isTLS` instead of `is_tls`?

Reply:
history reason